docs/development/reconciliation_en.md

```diff
@@ -0,0 +1,61 @@
+# How reconciliation works in the `k8s-dogu-operator`.
```

Comment:
The translation tool is not very good at handling broken paragraphs. It would be best to re-translate each paragraph without line breaks.